spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -625,6 +628,12 @@ class PodsProjectGenerator
                     bc.build_settings['TARGETED_DEVICE_FAMILY'].should == '1,2'
                   end
                 end
+
+                it 'sets the Swift version build setting when resources bundle contains sources and target has Swift' do
+                  @bundle_target.build_configurations.each do |bc|
+                    bc.build_settings['SWIFT_VERSION'].should == '3.2'
+                  end
+                end
```

Comment:
Should test that is isn't set when there aren't any compile sources, or is that covered by other tests?

Reply:
it is not but can do :D